### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by niedzielski**

### CODE REVIEW

**Comment by niedzielski** [comment]
:-1: Don't merge this guy yet. Still trying to figure out qqq mumbo jumbo.

**Comment by niedzielski** [comment]
:+1: ready 4 review

File: Wikipedia/Localizations/lv.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Messages for Latvian (latviešu)
```
Comment (montehurd) [inline Wikipedia/Localizations/lv.lproj/InfoPlist.strings]: @bgerstle hmm are we supposed to get InfoPlist.strings files?

File: Wikipedia/Localizations/ps.lproj/Localizable.strings [hunk 2]
```diff
@@ -73,6 +73,7 @@
 "main-menu-language-selection-saved" = "ژبټاکنه مو خوندي شوه";
 "main-menu-account-title-logged-out" = "گڼون";
 "main-menu-account-title-logged-in" = "ننوتي ياست لکه: $1";
```
Comment (montehurd) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: BUG! the $1 should not be reversed!

File: Wikipedia/Localizations/ps.lproj/Localizable.strings [hunk 3]
```diff
@@ -182,3 +183,5 @@
 "page-similar-titles" = "ورته مخونه";
 "captcha-reload" = "يو بل کپچا ښکاره کول";
 "captcha-prompt" = "له پورته انځور څخه کپچا متن ورکړئ";
+"home-more-like-footer" = "د $1 په څېر نور";
```
Comment (montehurd) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: Bug! See above.

Comment (montehurd) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: We need a test for these...

Comment (niedzielski) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: Will amend. We test for this on Android. I recommend it!

File: Wikipedia/Localizations/sd.lproj/Localizable.strings [hunk 4]
```diff
@@ -182,6 +184,8 @@
 "button-next" = "اڳيون";
 "button-done" = "ٿي ويو";
 "button-save" = "سانڍيو";
+"button-save-for-later" = "بعد لاءِ سانڍيو";
+"button-saved-for-later" = "بعد لاءِ سانڍل";
 "onboarding-create-account" = "کاتو کوليو";
 "onboarding-login" = "لاگ ان";
 "onboarding-already-have-account" = "اڳ ئي کاتو رکو ٿا؟ $1";
```
Comment (montehurd) [inline Wikipedia/Localizations/sd.lproj/Localizable.strings]: Dollar sign reversal bug!

File: Wikipedia/Localizations/sd.lproj/Localizable.strings [hunk 5]
```diff
@@ -250,3 +254,14 @@
 "page-issues" = "صفحي جا مسئلا";
 "page-similar-titles" = "ان جهڙا صفحا";
 "mwk-empty-title-error" = "خالي عنوان تي آپريشن پيش ڪرڻ ۾ ناڪام.";
+"home-title" = "تلاش ڪريو";
+"home-continue-reading-heading" = "پڙهڻ جاري رکو";
+"home-continue-related-heading" = "ڇو ته توهان پڙهو ٿا $1";
```
Comment (montehurd) [inline Wikipedia/Localizations/sd.lproj/Localizable.strings]: Bug.

File: Wikipedia/Localizations/sd.lproj/Localizable.strings [hunk 6]
```diff
@@ -250,3 +254,14 @@
...
+"home-more-like-footer" = "وڌيڪ $1 وانگر";
```
Comment (montehurd) [inline Wikipedia/Localizations/sd.lproj/Localizable.strings]: Bug.

File: Wikipedia/Localizations/lv.lproj/InfoPlist.strings (hunk 1 above)
Comment (bgerstle) [inline Wikipedia/Localizations/lv.lproj/InfoPlist.strings]: @montehurd yes, allows people to localize the app name

Comment (montehurd) [inline Wikipedia/Localizations/lv.lproj/InfoPlist.strings]: @bgerstle coolness

File: Wikipedia/Localizations/ps.lproj/Localizable.strings (hunk 3 above)
Comment (montehurd) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: @niedzielski is it, I wonder, just a single translator who is not aware these are never to be changed? Or do we need to add some sort of note to the TWN projects for this?

File: Wikipedia/Localizations/ps.lproj/Localizable.strings (hunk 2 above)
Comment (niedzielski) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: Done

File: Wikipedia/Localizations/ps.lproj/Localizable.strings (hunk 3 above)
Comment (niedzielski) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: Done

Comment (montehurd) [inline Wikipedia/Localizations/ps.lproj/Localizable.strings]: @niedzielski woohoo! thanks for the help! :)

File: Wikipedia/Localizations/azb.lproj/Localizable.strings [hunk 7]
```diff
@@ -45,12 +45,13 @@
 "search-recent-clear-cancel" = "وازگئچ";
 "search-recent-clear-delete-all" = "هامیسین سیل";
 "main-menu-title" = "آرتیق";
-"main-menu-language-title" = "ویکی‌پئدیادا 1$ آختار";
+"main-menu-language-title" = "ویکی‌پئدیادا $1 آختار";
```
Comment (montehurd) [inline Wikipedia/Localizations/azb.lproj/Localizable.strings]: @niedzielski Oops missed one.

Comment (niedzielski) [inline Wikipedia/Localizations/azb.lproj/Localizable.strings]: Hm, my sed script must not work in RTL. Will fix.

**Comment by niedzielski** [comment]
Here we go, I think. Atom to the rescue.

**Comment by bgerstle** [comment]
:+1: lgtm, can't find any reversed dollar signs in the diff. we'll try to add add'l verifications to the repo & travis for future cases

**Pull request merged**